felix/fv/tiered_policy_test.go

```diff
@@ -374,16 +401,394 @@ var _ = infrastructure.DatastoreDescribe("connectivity tests with policy tiers _
+					DstMeta:    host2_wl1_Meta,
+					DstService: flowlog.EmptyService,
+					Action:     "allow",
+					Reporter:   "dst",
+				},
+				FlowEnforcedPolicySet: flowlog.FlowPolicySet{
+					"0|tier1|default/tier1.np1-1|allow|0": {},
+				},
+				FlowPendingPolicySet: flowlog.FlowPolicySet{
+					"0|tier1|default/tier1.np1-1|allow|0": {},
+				},
+			})
+
+		// Flow logs expected to be seen only with tier default action set to Deny.
+		if endOfTierDrop {
+			flowTester.CheckFlow(
+				flowlog.FlowLog{
+					FlowMeta: flowlog.FlowMeta{
+						Tuple:      aggrTuple,
+						SrcMeta:    host1_wl1_Meta,
+						DstMeta:    host2_wl4_Meta,
+						DstService: flowlog.EmptyService,
+						Action:     "deny",
+						Reporter:   "dst",
+					},
+					FlowEnforcedPolicySet: flowlog.FlowPolicySet{
+						"0|tier1|tier1.ep2-4|deny|-1": {},
+					},
+					FlowPendingPolicySet: flowlog.FlowPolicySet{
+						"0|tier1|tier1.ep2-4|deny|-1": {},
+					},
+				})
+
+			flowTester.CheckFlow(
+				flowlog.FlowLog{
+					FlowMeta: flowlog.FlowMeta{
+						Tuple:      aggrTuple,
+						SrcMeta:    host2_wl4_Meta,
+						DstMeta:    host1_wl1_Meta,
+						DstService: flowlog.EmptyService,
+						Action:     "deny",
+						Reporter:   "src",
+					},
+					FlowEnforcedPolicySet: flowlog.FlowPolicySet{
+						"0|tier1|tier1.ep2-4|deny|-1": {},
+					},
+					FlowPendingPolicySet: flowlog.FlowPolicySet{
+						"0|tier1|tier1.ep2-4|deny|-1": {},
+					},
+				})
+		}
+
+		// Flow logs expected to be seen only with tier default action set to Pass.
+		if !endOfTierDrop {
+			flowTester.CheckFlow(
+				flowlog.FlowLog{
+					FlowMeta: flowlog.FlowMeta{
+						Tuple:      aggrTuple,
+						SrcMeta:    host2_wl4_Meta,
+						DstMeta:    host1_wl1_Meta,
+						DstService: flowlog.EmptyService,
+						Action:     "allow",
+						Reporter:   "src",
+					},
+					// Enforced and pending policy sets must be identical.
+					FlowEnforcedPolicySet: flowlog.FlowPolicySet{
+						"0|tier1|tier1.ep2-4|pass|-1":                   {},
+						"1|__PROFILE__|__PROFILE__.kns.default|allow|0": {},
+					},
+					FlowPendingPolicySet: flowlog.FlowPolicySet{
+						"0|tier1|tier1.ep2-4|pass|-1":                   {},
+						"1|__PROFILE__|__PROFILE__.kns.default|allow|0": {},
+					},
+				})
+
+			flowTester.CheckFlow(
+				flowlog.FlowLog{
+					FlowMeta: flowlog.FlowMeta{
+						Tuple:      aggrTuple,
+						SrcMeta:    host1_wl1_Meta,
+						DstMeta:    host2_wl4_Meta,
+						DstService: flowlog.EmptyService,
+						Action:     "allow",
+						Reporter:   "dst",
+					},
+					// Enforced and pending policy sets must be identical.
+					FlowEnforcedPolicySet: flowlog.FlowPolicySet{
+						"0|tier1|tier1.ep2-4|pass|-1":                   {},
+						"1|__PROFILE__|__PROFILE__.kns.default|allow|0": {},
+					},
+					FlowPendingPolicySet: flowlog.FlowPolicySet{
+						"0|tier1|tier1.ep2-4|pass|-1":                   {},
+						"1|__PROFILE__|__PROFILE__.kns.default|allow|0": {},
+					},
```

Comment:
This flow and the one before, are the ones actually check Felix generates flow logs for tier's default pass action.